bin/pip_find_builddeps.py

```diff
@@ -175,19 +182,24 @@ def main():
             "dependencies will be added if used in conjunction with -a/--append."
         ),
     )
+    ap.add_argument(
+        "--allow-binary",
+        action="store_true",
+        help="allow to download binary packages",
```

Comment:
I would go with something like "don't find build dependencies for packages that have wheels available for the current platform"